sdcm/cluster.py

```diff
@@ -1606,6 +1606,34 @@ def datacenter_setup(self, datacenters):
         cmd = cmd.format(datacenters[self.dc_idx], dc_suffix)
         self.remoter.run(cmd)
 
+    def patch_scylla_yaml(self, yaml_file=SCYLLA_YAML_PATH, **kwargs):
+        """
+        A user can send a dictionary of values ​​(just like the function "config_setup") and thus only change the
+        values he has chosen. If one of the keys contains a "None" value, the same value will be removed from the Yaml
+        file.
+        """
+        yaml_dst_path = os.path.join(tempfile.mkdtemp(prefix='scylla-longevity'), 'scylla.yaml')
+        with open(file=yaml_dst_path, mode='r') as scylla_yaml_file:
+            scylla_yml = yaml.safe_load(scylla_yaml_file)
+
+        for key, value in kwargs.items():
+            if value is None:
+                scylla_yml.pop(key, None)
+            else:
+                if scylla_yml.get(key, False):
```

Comment:
this would fail if the parameter is boolean and set to false,
```suggestion
                if scylla_yml.get(key, None) is None:
```

Reply:
Your'e right!!
Thanks